### PR TITLE
Goodbye websocket

### LINKED ISSUE #31: Kill the .slice(50) in FE because we will update to 100 locations

### LINKED ISSUE #42: Map as landing page

### PR DESCRIPTION
**Pull request by FelixSchwarzmeier**
closes #31 and closes #42

### CODE REVIEW

**Comment by muellerjonas** [comment]
closes #31 and #42?

File: src/App.tsx [hunk 1]
```diff
@@ -26,77 +29,57 @@ const theme = createMuiTheme({
...
-      const isIWrappedResult = (v: any): v is IWrappedResult =>
-        v.hasOwnProperty("new_val");
-
-      if (isIWrappedResult(data)) {
-        // incremental streamed data, @TODO: manage changes to existing users
-        dispatch("ADD_USERS", data.new_val);
-      } else {
-        // initial data load
+    let respStatus = 0;
+    fetch(`${utils.initRestEndpoint}`, { signal }).then((resp: Response) => { respStatus = resp.status; return resp.json(); }).then((data: IUser[]) => {
```
Comment (muellerjonas) [inline src/App.tsx]: Ah, der jQuery?

Comment (FelixSchwarzmeier) [reply src/App.tsx]: Haha nope, that are ES6 template-strings: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings

**Comment by muellerjonas** [comment]
Bye Bye Junimond

**Pull request merged**